dev-infrastructure/Makefile

```diff
@@ -31,29 +31,49 @@ lint:
 	done
 .PHONY: lint
 
-feature-registration:
+feature-registration: # hardcoded to eastus as this is a subscription deployment, not a resource group
 	@az deployment sub create \
-		--name "feature-registration"
+		--name "feature-registration" \
+		--location eastus \
 		--template-file templates/feature-registration.bicep \
 		--confirm-with-what-if
 .PHONY: feature-registration
 
-rg: feature-registration
```

Comment:
Puts the AFEC feature flags into "Registering" state which makes the AKS deployment fail.  

Just do this once per sub.